src/components/metadata/index.js

```diff
@@ -0,0 +1,248 @@
+
+  &:before {
+    content: '#';
+  }
+`
+
+const TagsSection = styled.div`
+  display: flex;
+  flex-wrap: wrap;
+  align-content: center;
+`
+
+const StyledA = styled.a`
+  text-decoration: none;
+`
+
+const authorPropType = PropTypes.shape({
+  id: PropTypes.string.isRequired,
+  name: PropTypes.string.isRequired,
+})
+
+class Metadata extends PureComponent {
+  static propTypes = {
+    categories: PropTypes.arrayOf(
+      PropTypes.shape({
+        id: PropTypes.string.isRequired,
+        name: PropTypes.string.isRequired,
+        sort_order: PropTypes.number.isRequired,
+      })
+    ),
+    tags: PropTypes.arrayOf(
+      PropTypes.shape({
+        id: PropTypes.string.isRequired,
+        name: PropTypes.string.isRequired,
+      })
+    ),
+    themeColor: PropTypes.string.isRequired,
+    writers: PropTypes.arrayOf(authorPropType),
+    photographers: PropTypes.arrayOf(authorPropType),
+    designers: PropTypes.arrayOf(authorPropType),
+    rawAutherText: PropTypes.string,
+  }
+
+  static defaultProps = {
+    categories: [],
+    tags: [],
+    writers: [],
+    photographers: [],
+    designers: [],
+    rawAutherText: '',
+  }
+
+  renderCategorySection() {
+    const { categories, themeColor } = this.props
+
+    _.sortBy(categories, ['sort_order'])
+
+    const categoriesJSX = _.map(categories, (cat, index) => {
+      return (
+        <CategoryFlex key={`category_${cat.id}`}>
+          <UpperSeparationLine />
+          <StyledA href={`/categories/${cat.id}`}>
+            <CategoryText
+              color={themeColor}
+              fontWeight={index === 0 ? 'bold' : 'normal'}
+            >
+              {cat.name}
+            </CategoryText>
+          </StyledA>
+        </CategoryFlex>
+      )
+    })
+
+    return <CategoryFlexBox>{categoriesJSX}</CategoryFlexBox>
+  }
+
+  renderTagsSection() {
+    const { tags } = this.props
+
+    const tagsJSX = _.map(tags, tag => {
+      return (
+        <StyledA key={`tag_${tag.id}`} href={`/tags/${tag.id}`}>
+          <TagButton>{tag.name}</TagButton>
+        </StyledA>
+      )
+    })
+
+    return (
+      <React.Fragment>
+        <TagsSection>{tagsJSX}</TagsSection>
+        <LowerSeparationLine />
+      </React.Fragment>
+    )
+  }
+
+  renderAuthorsRow(label, authors, themeColor) {
+    if (authors.length === 0) {
+      return null
+    }
+
+    const authorNamesJSX = _.map(authors, author => {
+      return (
+        <StyledA key={`author_${author.id}`} href={`/authors/${author.id}`}>
+          <AuthorName fontSize="16" color={themeColor} marginLeft="8">
+            {author.name}
+          </AuthorName>
+        </StyledA>
+      )
+    })
+
+    return (
+      <AuthorFlexBox>
+        <NoShrinkFlexItem>
+          <AuthorJobTitle fontSize="14" color="#808080" marginLeft="5">
+            {label}
+          </AuthorJobTitle>
+        </NoShrinkFlexItem>
+        <NoShrinkFlexItem>
+          <AngledSeparationLine color={themeColor} />
```

Comment:
The line seems should behind the job title on zepline

Reply:
Yeah, and it placed after job title already.